lib/broadway/batcher.ex

```diff
@@ -47,6 +48,10 @@ defmodule Broadway.Batcher do
 
   @impl true
   def handle_events(events, _from, state) do
+    metadata = %{name: state.name, events: events}
+    measurements = %{time: System.monotonic_time()}
+    :telemetry.execute([:broadway, :batcher], measurements, metadata)
```

Comment:
Not sure what idiomatic `event_name` should look like, but doesn't this need a 3rd atom to denote what kind of event is being produced by batcher? (`:start`/`:stop`)

Reply:
Yeah, we settled on :start/:stop on telemetry.